media-tv/kodi/kodi-17.3-r2.ebuild

```diff
@@ -0,0 +1,286 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Do not do multiple revbumps within a single PR unless you have a very good reason to do so. This is going to get pushed all at once anyway.